pkg/volume/volume.go

```diff
@@ -132,11 +132,39 @@ type Attacher interface {
 	Attach() error
 }
 
```

Comment:
Comment on `GetPath()` above:
```GO
type Volume interface {
	// GetPath returns the directory path the volume is mounted to.
	GetPath() string

	// MetricsProvider embeds methods for exposing metrics (e.g. used,available space).
	MetricsProvider
}
```

We need to make a clear distinction between the global mount path and the bind mount path. So let's break `GetPath()` in to `GetDeviceMountPath()` `GetPodBindMountPath()`